pkg/master/master.go

```diff
@@ -271,11 +277,11 @@ func New(c *Config) *Master {
 	}
 
 	// Select the first two valid IPs from portalNet to use as the master service portalIPs
-	serviceReadOnlyIP, err := service.GetIndexedIP(c.PortalNet, 1)
+	serviceReadOnlyIP, err := ipallocator.GetIndexedIP(c.PortalNet, 1)
 	if err != nil {
 		glog.Fatalf("Failed to generate service read-only IP for master service: %v", err)
 	}
-	serviceReadWriteIP, err := service.GetIndexedIP(c.PortalNet, 2)
+	serviceReadWriteIP, err := ipallocator.GetIndexedIP(c.PortalNet, 2)
```

Comment:
What's the benefit of hardcoding the kubernetes/kubernetes-ro service IPs vs just letting them be selected by the normal allocation method?

If we're concerned enough about them being deleted that we regularly ensure their existence, should we also be mildly concerned about another service being randomly allocated one of these IPs between when the kubernetes service gets deleted and re-created by the reconciler?